skills/emmaly-skills/skills/home-assistant/SKILL.md

```
@@ -60,21 +60,23 @@ WebSocket gives full access to commands not available via REST (dashboards, logs
 
 ### Connection URL
 
-Derive from `HASS_API_URL` — replace `http` with `ws` (or `https` with `wss`) and append `/api/websocket`:
+Derive from `HASS_API_URL` by replacing `http` with `ws` (or `https` with `wss`) and append `/api/websocket`:
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)